### PR TITLE
[MMI] Adds custodian icon next to metafox

### PR DESCRIPTION
**Pull request by albertolive**
## Explanation

Adds custodian icon next to metafox

image

https://consensyssoftware.atlassian.net/browse/MMI-2937

## Pre-merge author checklist

- [X] I've clearly explained:
  - [X] What problem this PR is solving
  - [X] How this problem was solved

## Pre-merge reviewer checklist

- [X] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [X] PR is linked to the appropriate GitHub issue

### CODE REVIEW

**Review by shane-t — DISMISSED** [review]

File: ui/components/ui/metafox-logo/metafox-logo.component.js [hunk 1]
```diff
@@ -11,12 +11,34 @@ export default class MetaFoxLogo extends PureComponent {
...
+        <img
+          {...iconProps}
+          src={custodyImgSrc}
+          className={classnames(
+            'app-header__custody-logo',
+            'app-header__custody-logo--icon',
+          )}
+          alt=""
+        />
+      </div>
```
Comment (DDDDDanica) [inline ui/components/ui/metafox-logo/metafox-logo.component.js:37]: we can remove div wrapper to keep only img tag as there's only one DOM
```suggestion
        <img
          {...iconProps}
          src={custodyImgSrc}
          className={classnames(
            'app-header__custody-logo',
            'app-header__custody-logo--icon',
          )}
          alt=""
        />
```

File: ui/components/app/app-header/app-header.component.js [hunk 2]
```diff
@@ -152,6 +159,10 @@ export default class AppHeader extends PureComponent {
                 }
                 history.push(DEFAULT_ROUTE);
               }}
+              ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+              custodyImgSrc={custodianIcon}
```
Comment (DDDDDanica) [inline ui/components/app/app-header/app-header.component.js:163]: Can we have 1 snapshot/dom snap insertion to cover this element in `ui/components/app/app-header/app-header.test.js`? :)

**Comment by DDDDDanica** [comment]
Hey @albertolive thanks for the hard work, I just wonder if you can attach a screenshot of DOM changes, and a screenshot of original ticket request (somehow i was not able to open in JIRA and i know most of MM teams are using zenhub) 
It would be super helpful for code review especially involving in some dom changes.
Sorry i didn't raise this one before for other MRs. 🙏

File: ui/components/app/app-header/app-header.component.js (hunk 2 above)
Comment (albertolive) [reply ui/components/app/app-header/app-header.component.js:163]: Added tests in MetaFox logo instead of app-header

**Review by DDDDDanica — APPROVED** [review]

**Review by pedronfigueiredo — APPROVED** [review]

**Pull request merged**